scripts/generate.js

```diff
@@ -0,0 +1,42 @@
+const prompts = require('prompts');
```

Comment:
if you're ok with a little bash, and don't want `prompts` as `devDep`, it could be merely:

```sh
#!/usr/bin/env bash

read -p "Does the new component have a specific mobile design (y/n)?" CHOICE

case "$CHOICE" in
  y|Y )
    npm run generate:component:mobile
  n|N )
    npm run generate:component
  * )
    echo "please choose y or n";;
esac
```

Reply:
Yes!
I wanted to use bash instead of node for this, but didn't know how.